applications/nrf5340_audio/tools/buildprog/program.py

```diff
@@ -41,31 +41,35 @@ def __populate_uicr(dev):
 def _program_cores(dev: DeviceConf) -> int:
     if dev.core_net_programmed == SelectFlags.TBD:
         if not path.isfile(dev.hex_path_net):
-            print("NET core hex not found. Built as APP core image.")
-            return 1
-
-        print(f"Programming net core on: {dev}")
-        cmd = (f"nrfjprog --program {dev.hex_path_net}  -f NRF53  -q "
-               f"--snr {dev.nrf5340_audio_dk_snr} --sectorerase --coprocessor CP_NETWORK")
-        ret_val = system(cmd)
-        if ret_val != 0:
-            if not dev.recover_on_fail:
-                dev.core_net_programmed = SelectFlags.FAIL
-            return ret_val
+            print(f"NET core hex not found. Built as APP core image. {dev.hex_path_net}")
```

Comment:
Should we add `dev.core_net_programmed = SelectFlags.FAIL`here?

Reply:
Change to now only building for app core.